docs/internal/result_handlers.md

```diff
@@ -0,0 +1,307 @@
+--result-handler=db-black-belts-i-dvt.pso_data_validator_results.results \
+2>/tmp/debug.log
+
+real	3m24.076s
+user	1m57.846s
+sys	0m3.508s
+```
+
+DVT stages:
+```
+04/03/2025 11:12:56 AM-DEBUG: Build config elapsed: 0.18s
+04/03/2025 11:13:00 AM-DEBUG: Target query elapsed: 4.44s
+04/03/2025 11:13:03 AM-DEBUG: Source query elapsed: 7.59s
+04/03/2025 11:13:19 AM-DEBUG: Generate report elapsed: 15.8s
+04/03/2025 11:16:18 AM-DEBUG: Write results to BigQuery elapsed: 178.15s
+```
+
+`vmstat` output:
+```
+procs -----------memory---------- ---swap-- -----io---- -system-- ------cpu-----
+ r  b   swpd   free   buff  cache   si   so    bi    bo   in   cs us sy id wa st
+ 0  0      0 32095248  21256 557508    0    0    21     8  375  500  9  1 90  0  0
+ 1  0      0 31426872  21264 557512    0    0     0    34 5278 5836 14  4 82  0  0
+ 1  0      0 30798496  21272 557512    0    0     0     1 1352 1849 21  4 75  0  0
+ 1  0      0 30601272  21272 557516    0    0     0     1 1261 1751 20  1 79  0  0
+ 1  0      0 30606352  21288 557544    0    0     0    12 1288 1723 14  0 86  0  0
+ 1  0      0 30608560  21304 557564    0    0     0     2 1309 1728 14  0 86  0  0
+ 0  0      0 30607392  21320 557588    0    0     0     8 1277 1751 13  0 87  0  0
+ 1  0      0 30606060  21336 557612    0    0     0     4 1237 1583 14  0 86  0  0
+ 1  0      0 30606124  21352 557668    0    0     3     4 1261 1573 14  0 85  0  0
+ 2  0      0 30606124  21368 557692    0    0     0    11 1241 1571 14  0 86  0  0
+ 1  0      0 30606092  21384 557712    0    0     0     5 1220 1571 14  0 86  0  0
+ 1  0      0 30605964  21400 557744    0    0     0    16 1190 1558 13  0 87  0  0
+ 0  0      0 30605932  21416 557756    0    0     0    11 1210 1566 14  0 86  0  0
+ 1  0      0 30606056  21432 557792    0    0     0     3 1221 1556 14  0 86  0  0
+ 0  0      0 30605048  21448 557824    0    0     0    16 1233 1584 14  0 86  0  0
+ 1  0      0 30604292  21464 557844    0    0     0     3 1204 1596 13  0 87  0  0
+ 1  0      0 30604292  21480 557868    0    0     0     2 1216 1569 14  0 86  0  0
+ 3  0      0 30603976  21504 557896    0    0     0    34 1226 1559 14  0 86  0  0
+ 1  0      0 30603880  21520 557924    0    0     0     2 1232 1567 14  0 86  0  0
+ 0  0      0 30603816  21536 557948    0    0     0     2 1194 1561 13  0 87  0  0
+ 1  0      0 30604352  21556 557968    0    0     0    14 1201 1558 13  0 87  0  0
+ 0  0      0 32091888  21572 557980    0    0     0     4 1108 1599  9  0 90  0  0
+ 0  0      0 32091888  21572 557980    0    0     0     0  853 1527  0  0 100  0  0
+```
+
+### PostgreSQL - Ibis client.insert
+
+Writing to PostgreSQL using the `insert()` method provided by Ibis.
+```python
+self._client.insert(table_name, result_df)
+```
+
+Command:
+```console
+time data-validation -ll DEBUG validate row -sc=oravol -tc=pg \
+-tbls=dvt_test.tab_vol_1m -pk=id --hash='*' \
+--result-handler=pg_rh.pso_data_validator_results.results \
```

Comment:
```suggestion
--result-handler=<GCP_PROJECT_ID>.pso_data_validator_results.results \
```

Reply:
`rg_rh` is the name of a connection file, not a project name. I've updated the doc.